### DISCUSSION TITLE
Welcome to SiPPOB Discussions!

### DISCUSSION THREAD

**Discussion by ZumaAkbarID** [ZumaAkbarID/SiPPOB#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZumaAkbarID** [comment]
Woi mau mulai kapan

**Reply by praditus343** [reply]
ayooo

**Comment by ZumaAkbarID** [comment]
@dicorzki @praditus343 @Syamsoelll @RachmasariAR @DennitaNF

**Reply by RachmasariAR** [reply]
gass

**Comment by praditus343** [comment]
hai, mau kapan?

aku ikut

**Comment by dicorzki** [comment]
p mabar

**Comment by praditus343** [comment]
Info info
